### ISSUE TITLE
Task: adding member (owner) to a Task does not work

### ISSUE THREAD

**Issue by bednar** [influxdata/influxdb#11491]
When I trying add member to the Task then the response is unsuccessful with an error:

```bash
-> POST 
-> http://127.0.0.1:9999/api/v2/tasks/034a72f16c10e000/members
-> {"id":"034a72f1741c6000"}

<- RESPONSE
<- {"code":"internal error","message":"unsupported resource type tasks"}
```

The problem is that the [FindResourceOrganizationID](https://github.com/influxdata/influxdb/blob/abe6ed3b8ba87bbc06bb72e722158602f487a40e/bolt/organization.go#L514) does not have implementation for the [TasksResourceType](https://github.com/influxdata/influxdb/blob/abe6ed3b8ba87bbc06bb72e722158602f487a40e/authz.go#L109).

**Comment by desa** [comment]
Note that #11701 does not solve this issue.

**Comment by jademcgough** [comment]
@bednar I'll deal with this now that #11664 has been merged, sorry to leave you hanging.

**Comment by bednar** [comment]
@jademcgough Thanks for that

**Comment by jademcgough** [comment]
Believe this was closed by #11741

**Comment by jademcgough** [comment]
Reopening until ^ has been merged
